content/actions/using-workflows/workflow-commands-for-github-actions.md

```diff
@@ -190,7 +190,7 @@ echo "::endgroup::"
 ::add-mask::{value}
 ```
 
-Masking a value prevents a string or variable from being printed in the log. Each masked word separated by whitespace is replaced with the `*` character. You can use an environment variable or string for the mask's `value`.
+Masking a value prevents a string or variable from being printed in the log. Each masked word separated by whitespace is replaced with the `*` character. You can use an environment variable or string for the mask's `value`. Masked outputs are considered as secrets, and will be redacted on the runner. See [Workflow commands for GitHub Actions](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions).
```

Comment:
Thanks for this clarification, @nikola-jokic. I think we can add a bit more explanation as to what being 'considered as secrets' means. Is this rewording accurate?

```suggestion
Masking a value prevents a string or variable from being printed in the log. Each masked word separated by whitespace is replaced with the `*` character. You can use an environment variable or string for the mask's `value`. When you mask a value, it is treated as a secret and will be redacted on the runner. For example, after you mask a value, you won't be able to set that value as an output.
```

Reply:
Hi @lucascosti, you are right. I am committing your changes. Thank you!